T/TurboPFor/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TurboPFor"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/powturbo/TurboPFor-Integer-Compression.git", "43fb0b2abaef27f6753f4494ffff638c3002f24c"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/TurboPFor-Integer-Compression
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/makefile.patch
+
+make -j${nprocs} libic.${dlext}
+mkdir -p ${libdir}
+mv libic.${dlext} ${libdir}
+
+install_license ${WORKSPACE}/srcdir/license.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# platforms = supported_platforms()
+platforms = [Linux(:x86_64)]
```

Comment:
With the changes you've applied you might be able to build this for all supported platforms

Reply:
Sure, let’s try it

Reply:
Doesn’t look like it’ll build on 32 bit, makes sense given that the library is all SIMD targeting recent CPUs